### PR TITLE
Queen Screech Rework

### PR DESCRIPTION
**Pull request by Surrealaser**
-Changes the Queen screech to be Halloss based.

-130 Halloss damage to adjacent enemies (just shy of pain-critting a full health marine), 70 at the edge (7 tiles away). Subject to tweaking.

-Stutter stun/knockdown; 1 second adjacent, 0.4  seconds at the edge.

-Visual distortions (screenshake and druggy overlay): 3 seconds + 10 times stun/knockdown

-Deafen duration: 20 times stun/knockdown.

-Hold orders reduce the stun/knockdown/deafen duration and Halloss damage by 10% per level.

-Removed snowflake earmuff immunity; it's a psychic scream.

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/Castes/Queen.dm [hunk 1]
```diff
@@ -356,18 +356,27 @@
 				shake_camera(M, 30, 1) //50 deciseconds, SORRY 5 seconds was way too long. 3 seconds now
 
 	for(var/mob/living/carbon/human/M in oview(7, src))
-		if(istype(M.wear_ear, /obj/item/clothing/ears/earmuffs))
-			continue
+		//if(istype(M.wear_ear, /obj/item/clothing/ears/earmuffs))
+		//	continue
```
Comment (ninjanomnom) [inline code/modules/mob/living/carbon/xenomorph/Castes/Queen.dm]: Just remove commented code

**Comment by Supermichael777** [comment]
mawp

**Comment by Surrealaser** [comment]
Well I removed the commented code; looks like this is ready for merge.

File: code/modules/mob/living/carbon/xenomorph/Castes/Queen.dm [hunk 2]
```diff
@@ -356,18 +356,22 @@
...
-		if(dist <= 4)
-			to_chat(M, "<span class='danger'>An ear-splitting guttural roar shakes the ground beneath your feet!</span>")
-			M.stunned += 4 //Seems the effect lasts between 3-8 seconds.
-			M.KnockDown(4)
+		var/reduction = max(1 - 0.1 * M.protection_aura, 0) //Hold orders will reduce the Halloss; 10% per rank.
+		var/halloss_damage = (max(0,140 - dist * 10)) * reduction //Max 130 beside Queen, 70 at the edge
+		var/stun_duration = max(0,1.1 - dist * 0.1) * reduction //Max 1 beside Queen, 0.4 at the edge.
+		
+		if(dist < 8)
+			to_chat(M, "<span class='danger'>An ear-splitting guttural roar shakes the ground beneath your feet and makes your world convulse!</span>")
```
Comment (ninjanomnom) [inline code/modules/mob/living/carbon/xenomorph/Castes/Queen.dm]: There's nothing in this that seems to indicate it's a psychic scream. Make it a bit more clear in the text

**Pull request merged**